app/javascript/lib/imperial2030.test.js

```diff
@@ -143,7 +143,7 @@ describe('imperial2030', () => {
         expect(game.nations.get(Nation2030.RU).treasury).toEqual(9);
       });
 
-      test('nations cannot be paid less than 0 if they control many units', () => {
+      test('nations lose money if they control many units', () => {
```

Comment:
Can nations lose money here? Does the last sentence from the rules here contradict this?
<img width="244" alt="Screenshot 2023-03-19 at 7 03 08 PM" src="https://user-images.githubusercontent.com/10481391/226215302-6ab648e5-d72f-413d-8889-7a2cff020920.png">